assets/src/setup/pages/save/index.js

```diff
@@ -212,15 +212,15 @@ export function Save() {
 									target="_blank"
 									rel="noreferrer"
 								>
-									{ __( 'Visit in Customizer', 'amp' ) }
+									{ __( 'Customize', 'amp' ) }
 								</Button>
 								<Button
 									isPrimary
 									href={ previewPermalink }
 									target="_blank"
 									rel="noreferrer"
 								>
-									{ __( 'Browse AMP Version', 'amp' ) }
+									{ __( 'Browse', 'amp' ) }
```

Comment:
Functionally, this is the same as the "Visit" button, so I suppose they could use the same text.